brigade-cr-gateway/cmd/brigade-cr-gateway/server.go

```diff
@@ -48,11 +48,18 @@ func newRouter(store storage.Store) *gin.Engine {
 	router := gin.New()
 	router.Use(gin.Recovery())
 
+	handler := webhook.NewDockerPushHook(store)
+
 	events := router.Group("/events")
 	{
 		events.Use(gin.Logger())
 
-		events.POST("/webhook/:org/:project/:commit", webhook.NewDockerPushHook(store).Handle)
+		// Of the form /webhook/brigade-123456789?commit=master
+		events.POST("/webhook/:org", handler.Handle)
```

Comment:
Should this `:org` be `:repo`? In this example it looks like you're skipping the org and just passing the repo / commit as a query param.

Or should this example be passing the org in the path, with the repo and commit as query params?

Reply:
Gin has a quirk where the positional params must have the same name. So even though it _is_ a repo ID, I have to name it `:org` to satisfy Gin's naming rules.

Reply:
Interesting - you might note that in a comment in the code. That was not obvious to me (but I don't know Gin very well 😄 ).

Reply:
it could be changed if we used a different handler function, but yeah, @technosophos is spot on.

This also threw me off for a second before I realized this was to work around that quirk. +1 on adding a comment to note that in this situation, org == repo.